Completed Task "Command pages: Role management" (#21)
MoonlightCapital/MoonlightBot-docs: role-management-commands/pause-role.md

# pause-role

> This command allows you to pause a role for a user for a specified duration. With a pause is meant that the role is
> removed from the user for the specified duration. After the duration has passed, the role will be reassigned to the
> user.

Reviewer: ```suggestion
This command allows you to pause a role for a user for a specified duration. This means that the role is
removed from the user for the specified duration. After the duration has passed, the role will be automatically reassigned to the
user.
```

### Options

* `user`: The user to pause the role for. You can mention the user or use their ID to specify the user.

> * `role`: The role to pause for the user. You can mention the role or use its ID to specify the role.

Reviewer: "you can..." parts are sort of unnecessary. With slash commands the user is directly guided into correct input by the UI

> * `duration`: The duration for which the role should be paused for the user. You can specify the duration in the

Reviewer: Explanation on duration formats is already in another page (see https://moonlightbot.gitbook.io/docs/start-up/arguments), putting it here us too verbose

  format `1w2d3h4m5s` where `w` stands for weeks, `d` for days, `h` for hours, `m` for minutes, and `s` for seconds.
  Note that you can omit any of the units if you don't want to specify them, know that you need to specify at least
  one unit.
* `reason`: The reason for pausing the role for the user. This is an optional parameter, which can be used for

>   moderation.

Reviewer: ```suggestion
* `reason`: The reason for pausing the role for the user. This is an optional parameter, which can be used for
  record-keeping.
```

### Options

* `user`: The user to pause the role for.
* `role`: The role to pause for the user.
* `duration`: The duration for which the role should be paused for the user. For more information on the duration

>   format, refer to the [duration format](../start-up/arguments.md#durations)

Reviewer: ```suggestion
* `user`: The user to pause the role for
* `role`: The role to pause for the user
* `duration`: The duration for which the role should be paused for the user. For more information on the
  format, refer to the [arguments page](../start-up/arguments.md#durations)
```

"duration format" (and more generically "duration" is repeated too many times in the paragraph. Also, I don't like bullet points ending with full stops (to change in other pages as well)
